File: program/program/convert-temperature-from-celsius-to-kelvin/ConvertTemperatureFromCelsiusToKelvin.ts
function celsiusToKelvin(celsius: number): number {
    const kelvin: number = celsius + 273.15;
    return kelvin;
}

const celsiusTemperature: number = -40; 
const kelvinTemperature: number = celsiusToKelvin(celsiusTemperature);
console.log(`${celsiusTemperature} degrees Celsius is equal to ${kelvinTemperature} Kelvin.`);

File: program/program/convert-temperature-from-celsius-to-kelvin/convert_temperature_from_celsius_to_kelvin.pl
sub celsius_to_kelvin {
    my ($celsius) = @_;
    my $kelvin = $celsius + 273.15;
    return $kelvin;
}

my $celsius_temperature = -40; 
my $kelvin_temperature = celsius_to_kelvin($celsius_temperature);
print "$celsius_temperature degrees Celsius is equal to $kelvin_temperature Kelvin.\n";
